src/hotspot/share/gc/shenandoah/shenandoahAsserts.cpp

```diff
@@ -210,6 +210,13 @@ void ShenandoahAsserts::assert_correct(void* interior_loc, oop obj, const char*
                   file,line);
   }
 
+  ShenandoahHeapRegion* obj_reg = heap->heap_region_containing(obj);
+  if (!heap->is_full_gc_move_in_progress() && !obj_reg->is_active()) {
```

Comment:
This is essentially ShenandoahHeap::is_in(), right? Might also want to check for obj < region->top() here? Or use is_in() to begin with?

Reply:
All right, true. We might just use `is_in` here.